apl-db-updater/src/main/resources/db/migration/h2/shard/V1_0__Initialize_tables.sql

```diff
@@ -56,8 +56,8 @@ CREATE TABLE IF NOT EXISTS `transaction`
 
 CREATE TABLE IF NOT EXISTS option
 (
-    `name`  VARCHAR(100) COLLATE utf8_bin not null,
-    `VALUE` VARCHAR(150) COLLATE utf8_bin DEFAULT NULL
+    `name`  VARCHAR(100) not null,
+    `VALUE` VARCHAR(150) DEFAULT NULL
```

Comment:
Index **option_name_value_idx** will not work without _COLLATE utf8_bin_ on MariaDb